src/BLSPubkeyRegistry.sol

```diff
@@ -126,6 +121,30 @@ contract BLSPubkeyRegistry is BLSPubkeyRegistryStorage {
         }
     }
 
+    function _processSingleQuorumApkUpdate(uint8 quorumNumber, BN254.G1Point memory point) internal {
+        uint256 historyLength = apkHistory[quorumNumber].length;
+        require(historyLength != 0, "BLSPubkeyRegistry._processQuorumApkUpdate: quorum does not exist");
```

Comment:
ngl this feels like unnecessary code duplication. just using a length-1 array ain't that bad IMO / not worth the risk of missing something in duplication (especially if we ever modify the code)